### ISSUE TITLE
Profile #1 : Unable to extract JSON data 

### ISSUE THREAD

**Issue by WallisonBreno** [pgrimaud/instagram-user-feed#291]
I've been getting this error when i use the getProfile function, has nothing to do with cache or authenticity of the account

**Comment by pgrimaud** [comment]
Hey, could you try to update to v6.15.0 version. Instagram made some changes and I (hopefully) fixed it this morning.

**Comment by RoseGoli** [comment]
hi

**Comment by RoseGoli** [comment]
hi
i have such same errors on posts data fetch from :

pgrimaud/instagram-user-feed/src/Instagram/Transport/AbstractDataFeed.php
Syntax error on line 61!

**Comment by pgrimaud** [comment]
> hi
> 
> i have such same errors on posts data fetch from :
> 
> 
> 
> pgrimaud/instagram-user-feed/src/Instagram/Transport/AbstractDataFeed.php
> 
> Syntax error on line 61!

Did you tried to upgrade to 6.15?

**Comment by RoseGoli** [comment]
yes i do

**Comment by RoseGoli** [comment]
> yes i do this the getBody result : for (;;);{"redirect":"https://www.instagram.com/p/Cc-wIYPDhUt/"}



> > hi
> > i have such same errors on posts data fetch from :
> > pgrimaud/instagram-user-feed/src/Instagram/Transport/AbstractDataFeed.php
> > Syntax error on line 61!
> 
> Did you tried to upgrade to 6.15?

yes i do
this the getBody result :
for (;;);{"redirect":"https://www.instagram.com/p/Cc-wIYPDhUt/"}

**Comment by andreifiroiu** [comment]
Yup, unfortunatelly Instagram is responding inconsistently when trying to get media data by calling instagram.com/p/[shorcode]?__a=1 (apparently based on login data and IP history). When isn't in the mood, will return:

for (;;);{"__ar":1,"error":1357004,"errorSummary":"Sorry, something went wrong","errorDescription":"Please try closing and re-opening your browser window.","payload":null,"hsrp":{"hblp":{"consistency":{"rev":1005640913}}},"lid":"7106408698253330412"}

**Comment by RoseGoli** [comment]
> Yup, unfortunatelly Instagram is responding inconsistently when trying to get media data by calling instagram.com/p/[shorcode]?__a=1 (apparently based on login data and IP history). When isn't in the mood, will return:
> 
> for (;;);{"__ar":1,"error":1357004,"errorSummary":"Sorry, something went wrong","errorDescription":"Please try closing and re-opening your browser window.","payload":null,"hsrp":{"hblp":{"consistency":{"rev":1005640913}}},"lid":"7106408698253330412"}

there is an other way like 'graphql/query' to get posts now
lib should be updated...

**Comment by pgrimaud** [comment]
> lib should be updated...

Feel free to open a PR 😄

**Comment by RoseGoli** [comment]
![image](https://user-images.githubusercontent.com/70085500/172823320-99587e40-bea4-446b-8567-6a34acedb204.png)
i dont really knwo how to use github but this is what i did to the last version of library and now its working just fine with graph api

**Comment by RoseGoli** [comment]
> > lib should be updated...
> 
> Feel free to open a PR 😄

hi there
i wanna ask if you have a telegram account/group where we can talk maybe?
it would be really cool to committee information in such platform...

**Comment by stifler97** [comment]
I get this error when trying to login. I have used this in local and remote server. The local server works but remote server fails.

**Comment by nam-co** [comment]
I was having the same problem and I came up with this, install openVPN or WireGuard to make your server work as a vpn, connect to the vpn from your computer and login to instagram using your browser, you verify/accept the new location, sounds complicated but actually it was very easy, after this I got it working 🤞

**Comment by Rubberg** [comment]
Still the same pb.
Method getProfile() returns code 200 with some html as datas (probably the instagram login page) causing a Syntax error in json_decode().

There are many issues opened for the same error :
https://github.com/pgrimaud/instagram-user-feed/issues/335
https://github.com/pgrimaud/instagram-user-feed/issues/318
https://github.com/pgrimaud/instagram-user-feed/issues/298
